src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -188,7 +188,7 @@ public void CopyTo(T[] array, int index)
 
             if (index < 0 || index > array.Length)
```

Comment:
So this works because if int is <0, the uint cast is a huge overflowed number right?
Does the performance benefit outweigh the slight lack of readability?